exercises/binary-search-tree/README.md

```diff
@@ -0,0 +1,52 @@
+Insert and search for numbers in a binary tree.
```

Comment:
This needs the title of the exercise as a heading: `# Binary Search Tree`

Reply:
Done